mirror/images.txt

```diff
@@ -6,6 +6,9 @@
 # format: <full-image-spec> <short-name>
 #
 
+# ceph-csi devel
```

Comment:
this is not needed, quay.io does not have pull limits like docker

remove the docker.io/ceph/ceph entry below.